docs/usage.md

```diff
@@ -32,34 +32,39 @@ document. However, for the moment we'll just use a pre-generated dataset:
 
 ```{code-cell} ipython3
 import zarr
-ds = zarr.open("_static/example_data.vcz")
+vcf_zarr = zarr.open("_static/example_data.vcz")
 ```
 
 This is what the genotypes stored in that datafile look like:
 
 ```{code-cell}
 :"tags": ["remove-input"]
 import numpy as np
-assert all(len(np.unique(a)) == len(a) for a in ds['variant_allele']) 
-assert any([np.sum(g) == 1 for g in ds['call_genotype']]) # at least one singleton
-assert any([np.sum(g) == 0 for g in ds['call_genotype']]) # at least one non-variable
-
-alleles = ds['variant_allele'][:].astype(str)
-sites = np.arange(ds['call_genotype'].shape[0])
-print(" " * 22, "Site:", " ".join(str(x) for x in range(8)), "\n")
-for sample in range(ds['call_genotype'].shape[1]):
-    for genome in range(ds['call_genotype'].shape[2]):
-        genotypes = ds['call_genotype'][:,sample, genome]
-        print(
-            f"Diploid sample {sample} (genome {genome}):",
-            " ".join(alleles[sites, genotypes])
-        )
+assert all(len(np.unique(a)) == len(a) for a in vcf_zarr['variant_allele']) 
```

Comment:
We need to be careful here not to give people patterns that will have poor performance on larger datasets. Elementwise iteration over zarr can have very poor performance, so you're nearly always better off getting the numpy array, like`` vcf_zarr['variant_allele'][:]``. I know it's weird, but it's better than having stuff running really slowly.

Reply:
Very happy to change this to the most efficient style. Maybe worth adding a comment about this in the code?